cmd/restic/cmd_backup.go

```diff
@@ -373,19 +420,34 @@ func collectTargets(opts BackupOptions, args []string) (targets []string, err er
 			if len(expanded) == 0 {
 				Warnf("pattern %q does not match any files, skipping\n", line)
 			}
-			lines = append(lines, expanded...)
+			targets = append(targets, expanded...)
+		}
+	}
+
+	for _, file := range opts.FilesFromVerbatim {
+		fromfile, err := readPatternsFromFile(file, false)
```

Comment:
`readPatternsFromFile()` always strips surrounding whitespace, but `-verbatim` should be just that, and not have this stripped. I guess add another boolean for that?

Reply:
Moved the whitespace trimming outside the function and extended the unit test.